tools/build/workspace.bzl

```diff
@@ -329,6 +331,7 @@ def _grpc_deps(locals):
         locals = locals,
         # on the master-with-bazel branch
         url = "https://boringssl.googlesource.com/boringssl/+archive/afc30d43eef92979b05776ec0963c9cede5fb80f.tar.gz",
+        sha256 = "54ff586a2bafbf3b7f9a9ca037f3f3490281061670aba2962a8431e9492f4b27"
```

Comment:
Builds are failing, apparently this should be:
1c5c3e596b286414d70cc97f3e9d4e0831d73ca33b6969ec57bf56cf35cd8a61
Can you double check?

Reply:
remove this, it unfortunately doesn't work, because GOB puts some sort of timestamps into the tar file, which means it changes every time it's downloaded.